packages/client/components/ScopePhaseArea.tsx

```diff
@@ -70,8 +70,7 @@ const ScopePhaseArea = (props: Props) => {
   const {meeting} = props
   const isDesktop = useBreakpoint(Breakpoint.SIDEBAR_LEFT)
   const {viewerMeetingMember} = meeting
-  if (!viewerMeetingMember) return null
-  const {user, teamMember} = viewerMeetingMember
+  const {user, teamMember} = viewerMeetingMember!
```

Comment:
-2 what if a user visits a meeting that they didn't join while it was active?
if this is about a lint rule you can just add a comment to ignore that rule for this file

Reply:
Yep, I was just trying to remove the lint rule. Change reverted 👍